modules/rest-root/src/main/java/org/elasticsearch/rest/root/MainResponse.java

```diff
@@ -95,6 +113,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             .field("lucene_version", version.luceneVersion().toString())
             .field("minimum_wire_compatibility_version", version.minimumCompatibilityVersion().toString())
```

Comment:
Should this be changed to `TransportVersion.MINIMUM_COMPATIBLE`?

Reply:
this is part of the public rest api. If we change this it would be a breaking change, right?

Reply:
It would just be changing one string into another...

Reply:
Sounds sensible